data/simpy.gram

```diff
@@ -27,9 +27,10 @@ compound_stmt[stmt_ty]: function_def | if_stmt | class_def | with_stmt | for_stm
 
 # NOTE: yield_expression may start with 'yield'; yield_expr must start with 'yield'
 assignment:
-    | !'lambda' a=target ':' b=expression c=['=' d=yield_expression { d }] {
-         _Py_AnnAssign(construct_assign_target(p, a), b, c,
-         a->kind == Name_kind ? 1 : 0, EXTRA) }
+    | !'lambda' '(' a=ann_assign_target ')' ':' b=expression c=['=' d=yield_expression { d }] {
```

Comment:
Question: Is there a way to join these two? Maybe something like:

`( '(' a=ann_assign_target ')' |  a=ann_assign_target )`

Reply:
I don't think so, since the action is different. When there is a parenthesis, the `simple` argument to `_Py_AnnAssign` must be 1.